desktop/app.js

```diff
@@ -58,7 +59,7 @@ module.exports = function main() {
       height: mainWindowState.height,
       minWidth: 370,
       minHeight: 520,
-      icon: iconPath,
+      icon: nativeImage.createFromPath(iconPath),
```

Comment:
According to the docs it looks like this should result in the same image as just passing the string path, as before.

> In Electron, for the APIs that take images, you can pass either file paths or NativeImage instances. An empty image will be used when null is passed. -[Electron API Docs](https://electronjs.org/docs/api/native-image)

Did this change make the icon work?